### ISSUE TITLE
Testing the Model on New Images. How can i add my own images to the trained model and comprare the accuracy.

### ISSUE THREAD

**Issue by HaRiZ0s** [avishkarsaha/translating-images-into-maps#10]
Dear @avishkarsaha ,

First of all, thanks for the excellent work and  sharing the code!

It was easy to set up the environment and train the model.

I am interested in testing the model on new images and check/compare the results. How can i add my own images to the trained model? Any kind of information about the dimensions and type of the images used for the model will be useful.

Thank you!

John

**Comment by wang-TJ-20** [comment]
@HaRiZ0s ，hello, can you share your train weights? I want to train the model, but my machine performance does not support it

**Comment by HaRiZ0s** [comment]
> @HaRiZ0s ，hello, can you share your train weights? I want to train the model, but my machine performance does not support it

[output.txt](https://github.com/avishkarsaha/translating-images-into-maps/files/9308213/output.txt)
[train_ious.txt](https://github.com/avishkarsaha/translating-images-into-maps/files/9308214/train_ious.txt)
[train_loss.txt](https://github.com/avishkarsaha/translating-images-into-maps/files/9308215/train_loss.txt)
[val_ious.txt](https://github.com/avishkarsaha/translating-images-into-maps/files/9308216/val_ious.txt)
[val_loss.txt](https://github.com/avishkarsaha/translating-images-into-maps/files/9308218/val_loss.txt)

**Comment by wang-TJ-20** [comment]
Thank you very much. Have you used the weight for inferenced? The author has not published the inference code. Now I have the weight, how can I inference?


------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "avishkarsaha/translating-images-into-maps"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年8月11日(星期四) 晚上7:31
***@***.***&gt;;
抄送:&nbsp;" ***@***.******@***.***&gt;;
主题:&nbsp;Re: [avishkarsaha/translating-images-into-maps] Testing the Model on New Images. How can i add my own images to the trained model and comprare the accuracy. (Issue #10)





  
@HaRiZ0s ，hello, can you share your train weights? I want to train the model, but my machine performance does not support it
  
output.txt
 train_ious.txt
 train_loss.txt
 val_ious.txt
 val_loss.txt

**Comment by wang-TJ-20** [comment]
@wang-TJ-20 hello, I have seen your val_loss.txt,. I find the val_loss  has not decreased. keep it 35 around.

**Comment by HaRiZ0s** [comment]
> @wang-TJ-20 hello, I have seen your val_loss.txt,. I find the val_loss has not decreased. keep it 35 around.

@wang-TJ-20 hello, yes you are right the val_loss hasnt decreased. I havent paid attention to the "experiment results". I am still trying to figure out how i can add my own images and make the visualization code work which @WilliamLucifer proposed in the other issue. I have my email attached in my github profile. If you want you can contact me so that we can help each other.

**Comment by wang-TJ-20** [comment]
this is my test.py


------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "avishkarsaha/translating-images-into-maps"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年8月21日(星期天) 凌晨2:34
***@***.***&gt;;
抄送:&nbsp;" ***@***.******@***.***&gt;;
主题:&nbsp;Re: [avishkarsaha/translating-images-into-maps] Testing the Model on New Images. How can i add my own images to the trained model and comprare the accuracy. (Issue #10)





  
@wang-TJ-20 hello, I have seen your val_loss.txt,. I find the val_loss has not decreased. keep it 35 around.
  
@wang-TJ-20 hello, yes you are right the val_loss hasnt decreased. I havent paid attention to the "experiment results". I am still trying to figure out how i can add my own images and make the visualization code work which @WilliamLucifer proposed in the other issue. I have my email attached in my github profile. If you want you can contact me so that we can help each other.

**Comment by HaRiZ0s** [comment]
@wang-TJ-20 

Can copy your test.py and share it as a .txt ?   Maybe .py files in comment section dont work.   You can also copy-paste your code such as the did in #4 visualization issue.

**Comment by wang-TJ-20** [comment]
@HaRiZ0s
[test.txt](https://github.com/avishkarsaha/translating-images-into-maps/files/9401422/test.txt)

**Comment by danfouer** [comment]
> > @wang-TJ-20 hello, I have seen your val_loss.txt,. I find the val_loss has not decreased. keep it 35 around.
> 
> @wang-TJ-20 hello, yes you are right the val_loss hasnt decreased. I havent paid attention to the "experiment results". I am still trying to figure out how i can add my own images and make the visualization code work which @WilliamLucifer proposed in the other issue. I have my email attached in my github profile. If you want you can contact me so that we can help each other.

I have the same Problem , the val_loss has not decreased

==> Validating on 15 minibatches

D:\ProgramData\Anaconda3\envs\TIIM\lib\site-packages\torch\utils\checkpoint.py:31: UserWarning: None of the inputs have requires_grad=True. Gradients will be None
  warnings.warn("None of the inputs have requires_grad=True. Gradients will be None")

**Comment by danfouer** [comment]
![inference_token104](https://user-images.githubusercontent.com/4875770/218233410-250efec4-c11b-48d3-baf0-00015656f368.png)
@wang-TJ-20 this is my result by your test.py, is that right?

**Comment by zacmilan** [comment]
> ![inference_token104](https://user-images.githubusercontent.com/4875770/218233410-250efec4-c11b-48d3-baf0-00015656f368.png) @wang-TJ-20 this is my result by your test.py, is that right?

This result looks bad..

**Comment by jiaxiangyangColin** [comment]
> ? Maybe .py files in comment section dont work

I have this problem, and do you know what the workround is?
Thanks!

**Comment by jiaxiangyangColin** [comment]
> > > @wang-TJ-20 hello, I have seen your val_loss.txt,. I find the val_loss has not decreased. keep it 35 around.
> > 
> > 
> > @wang-TJ-20 hello, yes you are right the val_loss hasnt decreased. I havent paid attention to the "experiment results". I am still trying to figure out how i can add my own images and make the visualization code work which @WilliamLucifer proposed in the other issue. I have my email attached in my github profile. If you want you can contact me so that we can help each other.
> 
> I have the same Problem , the val_loss has not decreased
> 
> ==> Validating on 15 minibatches
> 
> D:\ProgramData\Anaconda3\envs\TIIM\lib\site-packages\torch\utils\checkpoint.py:31: UserWarning: None of the inputs have requires_grad=True. Gradients will be None warnings.warn("None of the inputs have requires_grad=True. Gradients will be None")
hi,  have you solved this problem?

**Comment by handsomeZhuang** [comment]
> this is my test.py
> […](#)
> ------------------&nbsp;原始邮件&nbsp;------------------ 发件人: "avishkarsaha/translating-images-into-maps" ***@***.***&gt;; 发送时间:&nbsp;2022年8月21日(星期天) 凌晨2:34 ***@***.***&gt;; 抄送:&nbsp;" ***@***.******@***.***&gt;; 主题:&nbsp;Re: [avishkarsaha/translating-images-into-maps] Testing the Model on New Images. How can i add my own images to the trained model and comprare the accuracy. (Issue #10) @wang-TJ-20 hello, I have seen your val_loss.txt,. I find the val_loss has not decreased. keep it 35 around. @wang-TJ-20 hello, yes you are right the val_loss hasnt decreased. I havent paid attention to the "experiment results". I am still trying to figure out how i can add my own images and make the visualization code work which @WilliamLucifer proposed in the other issue. I have my email attached in my github profile. If you want you can contact me so that we can help each other. — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you were mentioned.Message ID: ***@***.***&gt;

hi，friend,
how to get the "lmdb" and "split" directory using for trained and test process?
thanks!

**Comment by handsomeZhuang** [comment]
> > @wang-TJ-20 hello, I have seen your val_loss.txt,. I find the val_loss has not decreased. keep it 35 around.
> 
> @wang-TJ-20 hello, yes you are right the val_loss hasnt decreased. I havent paid attention to the "experiment results". I am still trying to figure out how i can add my own images and make the visualization code work which @WilliamLucifer proposed in the other issue. I have my email attached in my github profile. If you want you can contact me so that we can help each other.

I have  the same problem! how to solve it?
